WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -182,14 +182,19 @@ class MySiteViewModel @Inject constructor(
                     visibleDynamicCards,
                     backupAvailable,
                     scanAvailable,
-                    cards
+                    cardsUpdate
             )
         } else {
             buildNoSiteState()
         }
         UiModel(currentAvatarUrl.orEmpty(), state)
     }
 
+    private fun CardsUpdate.handleCardsUpdateForSnackbarError() {
+        _onSnackbarMessage.postValue(Event(SnackbarMessageHolder(UiStringRes(R.string.my_site_dashboard_update_error))))
+        this.showSnackbarError = false
```

Comment:
I added it to prevent snackbar error from displaying multiple times, every time a partial state is updated. While this does not look like a perfect solution, I wasn't able to figure out any other working solution yet. Looking forward to hearing your thoughts.